tests/rgw/sanity_rgw_multisite.py

```diff
@@ -143,6 +143,15 @@ def run(**kw):
 
     verify_io_on_sites = config.get("verify-io-on-site", [])
     if verify_io_on_sites:
+        verify_out, err = test_site_node.exec_command(
+            cmd="sudo python3 "
+            + test_folder_path
+            + lib_dir
+            + "read_io_info.py -v False",
```

Comment:
The short option could be set as a `boolean` option. That is
```
sudo python 3 {test_folder_path}/{lib_dir}/read_io_info.py -v
```
So in this case, we avoid setting the short option (for False). For True, we use the above command.

Please ensure backward compatibility.